### ISSUE TITLE
test galaxy branch

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/tools-iuc#4015]

**Comment by bernt-matthias** [comment]
/run-all-tool-tests fork=bernt-matthias branch=topic/sam-input-accepts-bam

**Comment by blankenberg** [comment]
Is the branch pushed to github and public? 

https://github.com/bernt-matthias/tools-iuc/tree/topic/sam-input-accepts-bam shows not found to me.

**Comment by bernt-matthias** [comment]
> Is the branch pushed to github and public?
> 
> https://github.com/bernt-matthias/tools-iuc/tree/topic/sam-input-accepts-bam shows not found to me.

Yep, but its a branch of the galaxy repo: https://github.com/bernt-matthias/galaxy/tree/topic/sam-input-accepts-bam .. which should work.
